### DISCUSSION TITLE
Tailwind class doesn't work

### DISCUSSION THREAD

**Discussion by Surajj-00** [tailwindlabs/tailwindcss#13683] [Help]
I write **flex items-center justify-between** , flex & items-center work but justify-between doesn't work. 
Don't know why this happens.

![Screenshot 2024-05-15 140402](https://github.com/tailwindlabs/tailwindcss/assets/104910573/e338fcce-1466-48af-a6e1-f244884f7a40)

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- The `<div>` is of a width that is larger than the sum of the widths of the content such that `justify-between` would create discernable effect.

Otherwise, consider providing a project that reproduces the unexpected behavior.
